libraries/AC_PrecLand/AC_PrecLand_SITL.h

```diff
@@ -33,7 +33,7 @@ class AC_PrecLand_SITL : public AC_PrecLand_Backend
     uint32_t los_meas_time_ms() override { return _los_meas_time_ms; }
 
     // return true if there is a valid los measurement available
-    bool have_los_meas() override;
+    bool have_los_meas() override { return _have_los_meas; }
```

Comment:
what is the reason of this change of logic ?

Reply:
Hum
the change is good anyway was L39 is doing the calculation and we are still checking for update in less that 1s.